tests/xilinx/tribuf.ys

```diff
@@ -0,0 +1,11 @@
+read_verilog tribuf.v
+hierarchy -top top
+proc
+tribuf
+flatten
+synth
+equiv_opt -assert -map +/xilinx/cells_sim.v -map +/simcells.v synth_xilinx # equivalency check
+design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
+cd top # Constrain all select calls below inside the top module
+select -assert-count 1 t:$_TBUF_
```

Comment:
Comment about `$_DLATCH_*_` also applies here

Reply:
The same thoughts as with latches. Vivado synthesizes OBUFT cells. Maybe, I should report new issues to support this cases?

Reply:
An issue for BUFT cell support:
[#1388 ](https://github.com/YosysHQ/yosys/issues/1388)

Reply:
Please add issue number to comment

Reply:
Please add issue link to comment

Reply:
Resolved in #1420